bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -452,6 +453,46 @@ impl Room {
             Ok(())
         })
     }
+
+    /// Reports an event from the room.
+    ///
+    /// # Arguments
+    ///
+    /// * `event_id` - The ID of the event to report
+    ///
+    /// * `reason` - The reason for the event being reported (optional).
+    ///
+    /// * `score` - The score to rate this content as where -100 is most
+    ///   offensive and 0 is inoffensive (optional).
+    pub fn report_content(
+        &self,
+        event_id: String,
+        score: Option<i32>,
+        reason: Option<String>,
+    ) -> Result<()> {
+        let int_score: Option<Int>;
+        if let Some(value) = score {
+            int_score = Some(Int::from(value.clamp(-100, 0)));
+        } else {
+            int_score = None;
+        }
```

Comment:
I don't think we should `clamp` here, and instead let the server deal with the user-provided value however it sees fit. Also, this can be simplified to a use of `Option::map`:

```suggestion
        let int_score = score.map(|value| value.into());
```

Reply:
okay thanks for the tip, then I'll avoid clamping the value.